javascript/ql/src/Security/CWE-830/FunctionalityFromUntrustedSource.qhelp

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+	<overview>
+		<p>
+			Including a resource from an untrusted source or using an untrusted channel may
+			allow an attacker to include arbitrary code in the response.
+			When including an external resource (eg., a <code>script</code> element or an
+			<code>iframe</code> element) on a page, it is important to ensure that the received
+			data is not malicious.
+		</p>
+
+		<p>
+			When including external resources, it is possible to verify that the origin (the server
+			that responds to the request) is the intended one by using an <code>https</code> URL.
```

Comment:
It seems odd (to my way of thinking) to refer to the server that responds to a request as "the origin". Seem like the origin would be where the request originated.

Reply:
I meant "origin" as "origin of the resource" — but it's very ambiguous the way I'm using it here. Thank you. I'll fix it.